packages/ledger/src/ledger.canister.ts

```diff
@@ -1,35 +1,34 @@
-import { createServices, toNullable } from "@dfinity/utils";
+import type { QueryParams } from "@dfinity/utils";
+import { Canister, createServices, toNullable } from "@dfinity/utils";
 import type {
   BlockIndex,
   Tokens,
-  _SERVICE as SnsLedgerService,
+  _SERVICE as Icrc1LedgerService,
 } from "../candid/icrc1_ledger";
 import { idlFactory as certifiedIdlFactory } from "../candid/icrc1_ledger.certified.idl";
 import { idlFactory } from "../candid/icrc1_ledger.idl";
 import { toTransferArg } from "./converters/ledger.converters";
-import { SnsTransferError } from "./errors/ledger.errors";
-import { Canister } from "./services/canister";
-import type { SnsCanisterOptions } from "./types/canister.options";
+import { Icrc1TransferError } from "./errors/ledger.errors";
+import type { Icrc1LedgerCanisterOptions } from "./types/canister.options";
 import type { BalanceParams, TransferParams } from "./types/ledger.params";
-import type { SnsTokenMetadataResponse } from "./types/ledger.responses";
-import type { QueryParams } from "./types/query.params";
+import type { Icrc1TokenMetadataResponse } from "./types/ledger.responses";
 
-export class SnsLedgerCanister extends Canister<SnsLedgerService> {
-  static create(options: SnsCanisterOptions<SnsLedgerService>) {
+export class Icrc1LedgerCanister extends Canister<Icrc1LedgerService> {
```

Comment:
They are already working in icrc-2. Should we use another class to support that standard? Even though sometimes it might be the same canister?

Reply:
Good point. Given the fact that ICRC-2 is an extension of ICRC-1, I guess we can also in the future extend it. I'll rename the canister to `IcrcLedgerCanister`